File: New_HomeWork/4th/Bubble_sort.java
import java.util.*;
class Bubble_sort{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int[] Sort_List = new int[6];
        for(int k = 0; k < Sort_List.length; k++){
            Sort_List[k] = sc.nextInt();
        }
		int temp;
		for(int i = 0 ; i < Sort_List.length ; i ++) {
			for(int j = 0 ; j < Sort_List.length -i -1; j ++) {
				if(Sort_List[j]>Sort_List[j+1]) {
					temp = Sort_List[j];
					Sort_List[j] = Sort_List[j+1];
					Sort_List[j+1] = temp;
				}
			}
		}
		for(int i = 0 ; i < Sort_List.length ; i ++) {
			System.out.print(Sort_List[i] + " ");
		}
		sc.close();
    }
}

File: New_HomeWork/4th/Bubble_sort.py
Sort_List = list(map(int,input().split()))
for j in range(len(Sort_List)):
    for i in range(0, len(Sort_List)-j-1):
        if(Sort_List[i] > Sort_List[i+1]):
            temp = Sort_List[i]
            Sort_List[i] = Sort_List[i+1]
            Sort_List[i+1] = temp
print(Sort_List)